plans/rpminspect/README.md

```diff
@@ -0,0 +1,48 @@
+# Rpminspect
+
+<!-- SPHINX-START -->
+
+Run [rpminspect] on the current Copr project or Koji build
+
+## Synopsis
+
+```yaml
+plans:
+  import:
+    url: https://github.com/packit/tmt-plans
+    ref: main
+    name: /plans/rpminspect
+```
+
+## Options
+
+`RPMINSPECT_KOJI_BUILD`
+: Run `rpminspect` on the specified Koji build instead of the expected Copr project
+: Note: Not yet implemented
+
+## Examples
+
+- Inspect the upstream packit projects
+  ```yaml
+  plans:
+    import:
+      url: https://github.com/packit/tmt-plans
+      ref: main
+      name: /plans/rpminspect
+  ```
+- Inspect downstream koji builds
+  ```yaml
+  TBD
+  ```
+
+## See Also
+
+- Downstream Fedora-CI: [docker-runner][fedora-ci-docker], [tmt-plan][fedora-ci-tmt]
+- [rpminspect][rpminspect-doc]
+
+<!-- SPHINX-END -->
+
+[fedora-ci-docker]: https://github.com/fedora-ci/rpminspect-runner
+[fedora-ci-tmt]: https://github.com/fedora-ci/rpminspect-pipeline
```

Comment:
Is that a correct link? Or did you rather mean this?

* https://docs.fedoraproject.org/en-US/ci/tmt/

Reply:
About the `fedora-ci` links? I am linking there the equivalent project directly which I want to make it 1-to-1 compatible eventually. But good reference, I'll try to remember to add it in the top-level documentation